### DISCUSSION TITLE
I want it to start again at another time when the countdown is over. How can i do?

### DISCUSSION THREAD

**Discussion by ardaberrun** [ndresx/react-countdown#112] [Q&A] [answered]
For example i have  5 different time. 
```
  const [times,setTimes] = useState([
    {id:1,name:"Time 1",time: "08:27"},
    {id:2,name:"Time 2",time: "13:10"},
    {id:3,name:"Time 3",time: "15:47"},
    {id:4,name:"Time 4",time: "17:56"},
    {id:5,name:"Time 5",time: "19:48"}
  ])
```
When the time1 with now is over, I want to count now and time2 time without a break. I tried setting the time in the onComplete props but it didnt work.  In the renderer props I tried setting it up if complete and returning a new countdown but it didn't work too. How can i do this?

**Comment by ndresx** [comment, marked as answer]
Hey, you could maybe do something like this https://codesandbox.io/s/inspiring-lehmann-isv34?file=/index.js. I just changed the `time` values into integers so that they can be added easily to `Date.now()`.

**Reply by ardaberrun** [reply]
Thank you so much for helping. It worked
